website/docs/Software/OpenBCISoftware/02_GUI_Widget_Guide.md

```diff
@@ -255,6 +255,75 @@ Able to stream one data type. The data is sent sequentially, bit by bit, in cons
 
 User Datagram Protocol is used primarily for establishing low-latency and loss-tolerating connections between applications on the internet. The UDP protocol allows for up to three different data sets to be sent to three different applications.
 
+#### Running sample Python UDP scripts
+
+##### Sample scripts overview
+> 📌 **Repository URL**: [OpenBCI GUI Networking-Test-Kit - UDP](https://github.com/OpenBCI/OpenBCI_GUI/tree/master/Networking-Test-Kit/UDP)
+
+This section explains how to test **UDP marker reception** in OpenBCI GUI using the **Networking-Test-Kit**. The test consists of two Python scripts:
+
+- **`udp_receive_marker.py`** - Starts a UDP listener to receive marker packets.
+- **`udp_send_marker.py`** - Sends test UDP marker packets to the listener.
+
+
+##### Prerequisites
+
+Before running the test, ensure the following:
+
+- **Python 3** is installed (`python3 --version`).
+- You are inside the **Networking-Test-Kit/UDP** directory:
+
+  ```sh
+  cd path/to/OpenBCI_GUI/Networking-Test-Kit/UDP
+  ```
+
+##### Running the UDP Test
+> ⚠ **Important:** 
+> The **receiver must start before the sender**, otherwise the UDP packets will be lost.
+
+Open **Terminal 1** and run:
+
+```sh
+  python3 -m udp_receive_marker --ip 127.0.0.1 --port 12345 --option print
+```
+
+Open **Terminal 2** and run:
+
+```sh
+  python3 -m udp_send_marker --ip 127.0.0.1 --port 12345
+```
+
+##### Expected Sender Output
+```sh
+---------------------
+-- UDP MARKER SEND --
+---------------------
+IP: 127.0.0.1
+PORT: 12345
+--------------------=
+```
+
+##### Expected Receiver Output
+```sh
+--------------------
+-- UDP LISTENER --
+--------------------
+IP: 127.0.0.1
+PORT: 12345
+--------------------
+print option selected
+Listening for UDP packets...
+Received Marker: 2.506424330049714
+Received Marker: 1.5585284066901215
+Received Marker: 2.8304827346407384
+<continued...>
+```
+
+:::tip
+[The Marker Widget supports receiving UDP packets for marking timestamps in data via external hardware](#external-marking-via-udp)
```

Comment:
Doesn't just have to be external hardware. Could also be software that you want to sync with the recording